vernac/vernacinterp.ml

```diff
@@ -111,6 +111,13 @@ let with_fail ~st f =
     if not !Flags.quiet || !test_mode
```

Comment:
Should Succeed also check for `test_mode` whatever that is?

Reply:
That's so we can test the error message in output tests.
Since the success message doesn't change depending on the command there is no point checking it.